tools/common/lib.go

```diff
@@ -228,6 +231,10 @@ func CreateDestination(c *cli.Context, cClient ccli.Client, cliHelper common.Cli
 		ExitIfError(errors.New(strWrongZoneConfigCount))
 	}
 
+	if len(zoneConfigs.Configs) > 1 && unconsumedMessagesRetention < MinUnconsumedMessagesRetentionForMultiZoneDest {
```

Comment:
Can you do the same check within updateDestination() ? Also a comment for why this is necessary would be helpful.